### ISSUE TITLE
streamlink_for_tvh_container.sh: exited 1

### ISSUE THREAD

**Issue by SmokeyBR** [cgomesu/tvhlink#29]
today i tried to use one of the streams in tvheadend and i saw getting this when restarting tvheadend, can u confirm? its been a while i dont use this....

```
Author: cgomesu

Repo: https://github.com/cgomesu/tvhlink

***********************************************

[TVHlink] [info] Installing/upgrading Streamlink...

fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz

fetch http://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz

fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz

fetch http://dl-cdn.alpinelinux.org/alpine/v3.19/main/x86_64/APKINDEX.tar.gz

fetch http://dl-cdn.alpinelinux.org/alpine/v3.19/community/x86_64/APKINDEX.tar.gz

ERROR: unable to select packages:

  streamlink (no such package):

    required by: world[streamlink]

***********************************************

* Finished Streamlink install/upgrade script

* Message: APK: Critical error. Unable install required packages. Check previous messages.

***********************************************

[custom-init] streamlink_for_tvh_container.sh: exited 1
```

**Comment by cgomesu** [comment]
thanks for the heads up. the streamlink package was removed from Apine's package repository for some reason (https://pkgs.alpinelinux.org/packages). I think there's no one maintaining it anymore.  might need to revert the installation back to pip.  I'm kinda busy right now, so it might take a while for me to make this change. feel free to submit a pull request to fix this in the meantime.

**Comment by cgomesu** [comment]
in the meantime, you can manually fix as follows:

- enter the running container

  ```
  sudo docker exec -it tvheadend bash
  ```

- install pip and force install streamlink via pip system-wide

  ```
  apk add --upgrade --no-cache -X "$APK_MAIN" -X "$APK_COMMUNITY" python3 py3-pip
  pip install --break-system-packages -U pip setuptools
  pip install --break-system-packages -U streamlink
  ```

- exit the container and try running something

**Comment by SmokeyBR** [comment]
thank you, its working again

**Comment by jansohn** [comment]
the package was actually moved from testing to community: https://gitlab.alpinelinux.org/alpine/aports/-/commit/70c0ad63ed280c7f73bbab69c12e176377a5999b

It's available again starting from branch v3.21.

**Comment by cgomesu** [comment]
thanks for the heads up, @jansohn. don't remember seeing it listed in the official repo back them. in any case, installing streamlink via `pip` has been working pretty well, so I don't think I'll make any changes to the installation script right now.
